### PR TITLE
Update 124-flocking-boids.md

### PR DESCRIPTION
**Pull request by oxrock**
added flocking boids to contributions

### CODE REVIEW

File: _CodingChallenges/124-flocking-boids.md [hunk 1]
```diff
@@ -55,6 +55,12 @@ contributions:
       name: "Robin Andersson"
     url: "https://editor.p5js.org/RobinAndersson/sketches/r1-91uVxV"
     source: "https://editor.p5js.org/RobinAndersson/sketches/r1-91uVxV"
+  - title: "Flocking Boids"
+    author: oxrock
```
Comment (gruselhaus) [inline _CodingChallenges/124-flocking-boids.md]:
```suggestion
    author:
```

File: _CodingChallenges/124-flocking-boids.md [hunk 2]
```diff
@@ -55,6 +55,12 @@ contributions:
...
+    name: "oxrock"
```
Comment (gruselhaus) [inline _CodingChallenges/124-flocking-boids.md]:
``` suggestion
      name: "oxrock"
```

File: _CodingChallenges/124-flocking-boids.md (hunk 1 above)
Comment (gruselhaus) [inline _CodingChallenges/124-flocking-boids.md]:
```suggestion
    author:
```

File: _CodingChallenges/124-flocking-boids.md [hunk 3]
```diff
@@ -55,6 +55,12 @@ contributions:
...
+    url: "http://www.oxrockstudio.com/"
```
Comment (gruselhaus) [inline _CodingChallenges/124-flocking-boids.md]:
```suggestion
      url: "http://www.oxrockstudio.com/"
```

**Comment by shiffman** [comment]
Yay!

**Pull request merged**